pkg/node/node.go

```diff
@@ -461,7 +461,7 @@ func NewBee(
 
 		apiService.Mount()
 		apiService.SetProbe(probe)
-
+		apiService.SetWarmupTime(time.Now().Add(o.WarmupTime))
```

Comment:
here 's the tricky thing, if the postage batches are syncing, starting here, then setting this up here becomes inaccurate as we have not run this line yet https://github.com/ethersphere/bee/pull/5047/files#diff-515b2d0691202c4ea5e578ec896258ae9c440c381b608ed1eb7116a27847057dR782

maybe it would help if instead of passing a time to the api pkg, it can be done by toggling a bool externally?

Reply:
Now that I think of it again, the setter is better

Reply:
In the next PR we'll improve how we detect and set the warmup